### ISSUE TITLE
drop google analytics for Matomo on mochajs.org

### ISSUE THREAD

**Issue by boneskull** [mochajs/mocha#3349]
I've been using [Matomo](https://matomo.org/) for analytics on my personal blog, and am pretty impressed with it.

1. It doesn't own your data
2. It's open source
3. It's cheap if you don't want to self-host
4. We can make the analytics public
5. The analytics are comparable for our purposes

Currently, to see the mochajs.org analytics, you need a Google account, and I need to manually provide access within GA.  

So, I think we should swap it out.  If we need to pay, it will come out of the OpenCollective funds.

I can set this up, but wanted to know what others think?

cc @mochajs/core

**Comment by Bamieh** [comment]
@boneskull I personally hate the data ownership model of google analytics. So its a +1 for me even if this means going for a paid solution. I believe a good start is to list the metrics that we usually look at, and see if Matomo provides these metrics. If we decide to go with it, I am willing to set it up as well.

**Comment by outsideris** [comment]
Honestly, I like GA. :)
However, the feature to open the analytics publicly is fantastic and it is fit for our project.

Agreed!

**Comment by Munter** [comment]
I've never accessed the GA data, so it's not a loss for me personally. I like moving that visitor data out of Googles hands, and especially the feature about making some data publicly available if we want to

**Comment by boneskull** [comment]
I don't think we "usually" look at any metrics; but this is just good information to have.  The only things GA would have and Matomo wouldn't are likely very advanced or simply hidden behind a higher price point.  We don't need any of that stuff.  Even the cheapest package can be used for e-commerce, which is much more involved than what we're after.
